superset/assets/src/dashboard/util/publishSubscriberUtil.js

```diff
@@ -0,0 +1,48 @@
+import * as _ from 'lodash';
+
+export const APPLY_FILTER = 'APPLY_FILTER';
+export const USE_AS_MODAL = 'USE_AS_MODAL';
+const CHART_UPDATE_SUCCEEDED = "CHART_UPDATE_SUCCEEDED"
+//check slice is modal or not
+export function isModalSlice(slice) {
```

Comment:
can we rename this function to "isSliceUsedAsModal" or something more descriptive

Reply:
i think it is short and descriptive..

Reply:
I also think this is descriptive enough